packages/editor/src/components/collab-sidebar/index.js

```diff
@@ -237,12 +282,19 @@ export default function CollabSidebar() {
 	const { getActiveComplementaryArea } = useSelect( interfaceStore );
 	const isLargeViewport = useViewportMatch( 'medium' );
 
-	const { postId } = useSelect( ( select ) => {
-		const { getCurrentPostId } = select( editorStore );
+	const { postId, postType } = useSelect( ( select ) => {
+		const { getCurrentPostId, getCurrentPostType } = select( editorStore );
+		const _postId = getCurrentPostId();
+		const _postType = getCurrentPostType();
+
 		return {
-			postId: getCurrentPostId(),
+			postId: _postId,
+			postType: _postType,
 		};
 	}, [] );
+	const [ blocks ] = useEntityBlockEditor( 'postType', postType, {
```

Comment:
@Mamaduka should we return the commentId->block mapping (for the popover anchor) from useBlockComments so we don't have to add it here?